### ISSUE TITLE
Loss does not goes down

### ISSUE THREAD

**Issue by gorkem7** [shaoshengsong/MobileNetV3-SSD#22]
Hello, 
Thank you for your repo. 
I have used your pretrained model, trained it one more time with VOC dataset for 100 epochs but Loss does not go less than 4.8, and the model accuracy is not good at all. I have used VOC 2012 and 2007 dataset for training and 2007 for testing. Do you have any idea what it could be happened?
Regards

**Comment by shaoshengsong** [comment]
It is recommended that  git clone this version 
https://github.com/shaoshengsong/MobileNetV3-SSD-Compact-Version
